lib/pluginmanager/pack_command.rb

```diff
@@ -10,4 +10,13 @@ def archive_manager
   def file_extension
     zip? ? ".zip" : ".tar.gz"
   end
+
+  def signal_deprecation_warning_for_pack
+  message =<<-EOS
+The pack and the unpack command are now deprecated and will be removed in a future version of Logstash,
+See the `prepare-offline-pack` to replace your workflow, you can get the documentation by running `bin/logstash-plugin prepare-offline-pack --help` or more detailled examples with `bin/logstash-plugin prepare-offline-pack`
```

Comment:
See the `prepare-offline-pack` to update your workflow. You can get documentation about this by running `bin/logstash-plugin prepare-offline-pack --help`